flux/post_install.md

```diff
@@ -1,7 +1,7 @@
 ## Flux
 
 Flux is a Continuous Delivery solution for Kubernetes.
-Flux is constructed with the [GitOps Toolkit](https://toolkit.fluxcd.io/components/),
```

Comment:
Flux ones are already committed via your other PR, can you stash these commits ?

Reply:
Yeah, my mistake...Will remove the them.